.travis.yml

```diff
@@ -0,0 +1,14 @@
+language:      node_js
```

Comment:
I don't believe we currently have travis enabled. Please make a new pull request for travis specifically, if you'd like to add CI.